pkg/standalone-utils/contracts/EclpLPOracle.sol

```diff
@@ -0,0 +1,110 @@
+// SPDX-License-Identifier: LicenseRef-Gyro-1.0
+
+pragma solidity ^0.8.24;
+
+import { AggregatorV3Interface } from "@chainlink/contracts/src/v0.8/shared/interfaces/AggregatorV3Interface.sol";
+import { SafeCast } from "@openzeppelin/contracts/utils/math/SafeCast.sol";
+
+import { ILPOracleBase } from "@balancer-labs/v3-interfaces/contracts/standalone-utils/ILPOracleBase.sol";
+import { IGyroECLPPool } from "@balancer-labs/v3-interfaces/contracts/pool-gyro/IGyroECLPPool.sol";
+import { Rounding } from "@balancer-labs/v3-interfaces/contracts/vault/VaultTypes.sol";
+import { IBasePool } from "@balancer-labs/v3-interfaces/contracts/vault/IBasePool.sol";
+import { IVault } from "@balancer-labs/v3-interfaces/contracts/vault/IVault.sol";
+
+import { SignedFixedPoint } from "@balancer-labs/v3-pool-gyro/contracts/lib/SignedFixedPoint.sol";
+import { FixedPoint } from "@balancer-labs/v3-solidity-utils/contracts/math/FixedPoint.sol";
+import { GyroECLPMath } from "@balancer-labs/v3-pool-gyro/contracts/lib/GyroECLPMath.sol";
+import { GyroECLPPool } from "@balancer-labs/v3-pool-gyro/contracts/GyroECLPPool.sol";
+
+import { LPOracleBase } from "./LPOracleBase.sol";
+
+/**
+ * @notice Oracle for E-CLP pools.
+ */
+contract EclpLPOracle is LPOracleBase {
+    using FixedPoint for uint256;
+    using SignedFixedPoint for int256;
+    using SafeCast for *;
+
+    uint256 private constant _ONEHALF = 0.5e18;
```

Comment:
```suggestion
```
Unused here. There is a definition of this in ECLPMath; should be _ONE_HALF (two words).